language-support/ts/codegen/tests/ts/generated/src/__tests__/test.ts

```diff
@@ -12,15 +12,20 @@ import * as LibMod from '../daml/daml-tests/Lib/Mod';
 const LEDGER_ID = 'daml2ts-tests';
```

Comment:
The tests in here could use some cleanup. I'll do that in a separate PR once this one has landed.